tests/test_instr.py

```diff
@@ -156,6 +159,41 @@ def test_invalid_arg(self):
         label = Label()
         block = BasicBlock()
 
+        if PY314:
```

Comment:
Since I define dedicated groups of opcode which are version dependent (such as BITFLAG_OPCODES) I would prefer for the tests to use those to avoid duplicating the version checks. Otherwise LGTM (and sorry for the long silence)

Reply:
Sure np I'll make the change

Reply:
okay I changed to iteration over the opcode groups (like I saw that you do for `BITFLAG_OPCODES`. It's not super "elegant" because I don't know if you plan to add more opcode to the respective groups...